github-actions/src/main/scala/org/typelevel/sbt/gha/JavaSpec.scala

```diff
@@ -28,19 +28,25 @@ object JavaSpec {
   def temurin(version: String): JavaSpec = JavaSpec(Distribution.Temurin, version)
   def corretto(version: String): JavaSpec = JavaSpec(Distribution.Corretto, version)
   def graalvm(version: String): JavaSpec = JavaSpec(Distribution.GraalVM, version)
+  @deprecated("Migrate to `temurin`or `semeru` instead.", "0.5.0")
   def openj9(version: String): JavaSpec = JavaSpec(Distribution.OpenJ9, version)
   def oracle(version: String): JavaSpec = JavaSpec(Distribution.Oracle, version)
+  def semeru(version: String): JavaSpec = JavaSpec(Distribution.Semeru, version)
+  def microsoft(version: String): JavaSpec = JavaSpec(Distribution.Microsoft, version)
 
   sealed abstract class Distribution(val rendering: String) extends Product with Serializable
 
   object Distribution {
     case object Temurin extends Distribution("temurin")
     case object Corretto extends Distribution("corretto")
+    @deprecated("Migrate to `temurin`or `semeru` instead.", "0.5.0")
```

Comment:
Isn't OpenJ9 specifically succeeded by Semeru?

>  It is highly recommended to migrate workflows from adopt and adopt-openj9, to temurin and semeru respectively

I think `adopt` -> `temurin`, `adopt-openj9` -> `semeru`.

https://github.com/actions/setup-java#supported-distributions

Reply:
You are right 👍